dd-java-agent/appsec/src/main/java/com/datadog/appsec/powerwaf/PowerWAFModule.java

```diff
@@ -410,23 +411,26 @@ public void onDataAvailable(
         log.debug("Skipped; the WAF is not configured");
         return;
       }
-      try {
-        StandardizedLogging.executingWAF(log);
-        long start = 0L;
-        if (log.isDebugEnabled()) {
-          start = System.currentTimeMillis();
-        }
 
-        resultWithData = doRunPowerwaf(reqCtx, newData, ctxAndAddr, isTransient);
+      StandardizedLogging.executingWAF(log);
+      long start = 0L;
+      if (log.isDebugEnabled()) {
+        start = System.currentTimeMillis();
+      }
 
+      try {
+        resultWithData = doRunPowerwaf(reqCtx, newData, ctxAndAddr, isTransient);
+      } catch (TimeoutPowerwafException tpe) {
+        log.info("Timeout calling the WAF", tpe);
```

Comment:
```suggestion
        log.debug("Timeout calling the WAF", tpe);
```

@cataphract Let's reduce timeouts to debug level, since they might be spammy in some cases and will be better handled by metrics.